layouts/_default/content.html

```diff
@@ -1,7 +1,24 @@
 <div class="td-content">
   <h1>{{ .Title }}</h1>
   {{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
-  {{ .Content }}
+  {{ $out := .Content}}
```

Comment:
Did you also test the local build too? Early on there was a lot of discrepancies between how Netlify built using Hugo and how the default local Hugo server built our content, thus the move to "building" the same way in both places.

Reply:
Yes, I've been using local build to test.